src/main/scala/org/broadinstitute/hail/expr/AST.scala

```diff
@@ -1737,3 +1737,26 @@ case class If(pos: Position, cond: AST, thenTree: AST, elseTree: AST)
     }
   }
 }
+
+case class Splat(pos: Position, lhs: AST) extends AST(pos, lhs) {
+  override def typecheckThis(): BaseType = {
+    lhs.`type` match {
+      case t: TStruct => TSplat(t)
+      case t => parseError(
+        s"""splatting ( <identifier>.* ) operations are only supported on `Struct'
+            |  Found `$t'
+         """.stripMargin)
+    }
+  }
+
+  override def eval(ec: EvalContext): () => Any = {
+    val nElem = `type`.asInstanceOf[TSplat].struct.size
```

Comment:
While this seems fine, I think it is simpler to have the grammar for export args be [id =] expr [.*] and have the return value include an extra "splat" boolean instead of creating a new type.  I'm primarily worried about error checking.  TSplat can show up anywhere now.  What happens if I `annotatevariants -c 'va.splat = va.info.*'`?